docs/LEARNING.md

```diff
@@ -0,0 +1 @@
+Please see the "Useful Haskell Resources" tab. 
```

Comment:
Learning should be about learning the language from scratch, whereas resources is more about tools, references, and resources that you might use while developing in Haskell.